upstream-community-operators/seldon-operator/seldonoperator.crd.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: seldondeployments.machinelearning.seldon.io
+  name: seldondeployments.seldon
```

Comment:
seldondeployments.seldon" is invalid: [spec.group: Invalid value: "seldon": should be a domain with at least one dot.
metadata:name: Use as a name for the CustomResourceDefinition. It must use the {plural-name}.{group} format and use the values from the group and plural fields. If you do not follow these rules, you receive a validation error when installing the CRD.
Read here: https://github.com/kyma-project/community/blob/master/guidelines/custom-resource-definitions.md#location-and-file-name